scripts/ioc-deploy

```diff
@@ -0,0 +1 @@
+ioc_deploy.py
```

Comment:
Doesn't this need a `python ioc_deploy.py` or something?

Reply:
This is how github renders a symbolic link

Reply:
oh I see.  It's very similar to the other bash scripts that exist in engineering-tools with the same pattern